tests/unit_tests/common.py

```
@@ -0,0 +1,209 @@
+import os
+import pytest
+import shutil
+
+from common.tools.read_file import read_file
+from common.tools.list_files import list_files
+from common.tools.create_directory import create_directory
+from common.tools.create_file import create_file
+from common.tools.summarize import summarize
```

Ruff (0.11.9)

`common.tools.summarize.summarize` imported but unused

Remove unused import: `common.tools.summarize.summarize`

(F401)